### PR TITLE
feat: setup repo as NPM package with edX standards.

### PR DESCRIPTION
**Pull request by kenclary**

### CODE REVIEW

File: openedx.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# openedx.yaml
+
+---
+owner: edx/fedx-team
```
Comment (davidjoy) [inline openedx.yaml:4]: May want to change this to T&L, though I'm not sure if it is used/matters.

**Review by davidjoy — APPROVED** [review]
LGTM 🚢

File: src/setupTest.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+process.env.MARKETING_SITE_BASE_URL = 'http://localhost:18000';
+process.env.ORDER_HISTORY_URL = 'localhost:1996/orders';
+process.env.REFRESH_ACCESS_TOKEN_ENDPOINT = 'http://localhost:18000/login_refresh';
+process.env.SEGMENT_KEY = 'segment_whoa';
+process.env.SITE_NAME = 'edX';
+process.env.USER_INFO_COOKIE_NAME = 'edx-user-info';
+process.env.LOGO_URL = 'https://edx-cdn.org/v3/default/logo.svg';
+process.env.LOGO_TRADEMARK_URL = 'https://edx-cdn.org/v3/default/logo-trademark.svg';
+process.env.LOGO_WHITE_URL = 'https://edx-cdn.org/v3/default/logo-white.svg';
+process.env.FAVICON_URL = 'https://edx-cdn.org/v3/default/favicon.ico';
```
Comment (doctoryes) [inline src/setupTest.js:29]: NIT: I compared this list to the one here:
https://github.com/edx/frontend-platform/blob/master/.env.development
and there appear to be some missing values. Not sure if it's important at all.

**Review by doctoryes — APPROVED** [review]
Just a question/nit.

File: src/setupTest.js (hunk 2 above)
Comment (kenclary) [reply src/setupTest.js:29]: these files were based on the configuration of frontend-component-header, which is a recent example of a frontend library.

**Pull request merged**